src/app/faq/faq.html

```diff
@@ -1,68 +1,15 @@
-      Very soon. Meanwhile, please sign up to our email list to get the latest news, discounts and be the first to know when we launch.
-    </div>
-
-    <div uib-accordion-group class="panel-default" is-open='menuStatus[4].isOpen'>
-      <uib-accordion-heading>
-        Does HelloBible align with my beliefs? <i class="pull-right glyphicon" ng-class="{'glyphicon-chevron-down': menuStatus[4].isOpen , 'glyphicon-chevron-right': !menuStatus[4].isOpen }"></i>
-      </uib-accordion-heading>
-      Our content is Bible based. We focus on Scripture and we share suggestions on how to teach Bible Stories to children. We hope that those ideas and suggestions apply to a broad range of christian families.
-      If you have any questions please do not hesitate to contact us.
-    </div>
-
-    <div uib-accordion-group class="panel-default" is-open='menuStatus[5].isOpen'>
-      <uib-accordion-heading>
-        Who is Danny the Donkey? <i class="pull-right glyphicon" ng-class="{'glyphicon-chevron-down': menuStatus[5].isOpen , 'glyphicon-chevron-right': !menuStatus[5].isOpen }"></i>
-      </uib-accordion-heading>
-      Danny the Donkey is our little helper who takes the kids on their journey through the Bible. He welcomes, explains the crafts and knows a lot of great Bible Verses.
-
-      Donkeys are humble and often serving behind the scenes. In the Bible donkeys are constantly mentioned and considered reliable helpers to the humans. The New Testament tells us that Jesus entered Jerusalem on a donkey while the crowd worshipped him. Why did Jesus ride on an unimportant donkey instead of a mighty horse in such an important occasion? This shows that Jesus is a humble king, nobody needs to be perfect in order to serve Jesus.
-
-      Let's all be reminded by Danny the Donkey, that we are important to Jesus and that he wants to use us.
-    </div>
-
-    <div uib-accordion-group class="panel-default" is-open='menuStatus[6].isOpen'>
-      <uib-accordion-heading>
-        How can I support HelloBible? <i class="pull-right glyphicon" ng-class="{'glyphicon-chevron-down': menuStatus[6].isOpen , 'glyphicon-chevron-right': !menuStatus[6].isOpen }"></i>
-      </uib-accordion-heading>
-      You like HelloBible? There are many ways to support us.
-      <ul>
-        <li>Pray for us!</li>
-        <li>Tell your family and friends about HelloBible.</li>
-        <li>Visit us on Facebook</li>
-        <li>Visit us on Pinterest</li>
-        <li>Contact us for feedback at info@hellobible.org. We would love to hear about your ideas and proposals.</li>
-      </ul>
+<div ng-controller="FaqController as faqCtrl" class="__faq-accordion row">
+  <uib-accordion close-others="true" class="col-sm-10 col-sm-offset-1 col-md-8 col-md-offset-2">
+
+    <div uib-accordion-group is-open="isopen" class="panel-default __faq-accordion-item" ng-repeat="item in faqCtrl.faq">
+      <div ng-repeat="x in item.content">
+        <uib-accordion-heading ng-click="isopen=!isopen">
```

Comment:
We should use a function in the controller that is called on each click. It should take the title of the item as parameter. Within that function, we should track a google analytics event as we do in the pricing controller (category: FAQ, action: open, label: the title).